tests/test-profile.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect, devices } from '@playwright/test';
+
+test.use({
+  storageState: 'playwright/.auth/test-user-with-data.json',
+});
+
+test.use({
+  ...devices['iPhone 14 Pro'],
+});
+
+test('test-edit-own-profile', async ({ page }) => {
+  await page.goto('http://localhost:8100/home/profile');
+  await page.getByRole('button', { name: 'Edit Profile' }).click();
+  await page.getByLabel('First Name').fill('First Name');
+  await page.getByLabel('Last Name').fill('Second Name');
+  await page.getByLabel('BioChallenge me with your').fill('Changing up Bio for testing');
```

Comment:
I think this is a typo
```suggestion
  await page.getByLabel('Bio').fill('Changing up Bio for testing');
```

Reply:
@v4lentin1879